cli/app.go

```diff
@@ -456,6 +465,63 @@ var app = &cli.App{
 		},
 	},
 	Commands: []*cli.Command{
+		{
+			Name:  "traces",
+			Usage: "Work with viam-server traces",
+			Subcommands: []*cli.Command{
+				{
+					Name:      "import-local",
+					Usage:     "Import traces from a local viam server trace file to an OTLP endpoint.",
+					ArgsUsage: "<traces file>",
+					Flags:     commonOtlpFlags,
+					Action:    createCommandWithT(traceImportLocalAction),
+				},
+				{
+					Name: "import-remote",
+					Description: `
+In order to use the import-remote command, the machine must have a valid shell type service.
+Organization and location are required flags if using name (rather than ID) for the part.
+Note: There is no progress meter while copying is in progress.
+`,
+					Usage: "Import traces from a remote viam machine to an OTLP endpoint.",
+					Flags: lo.Flatten([][]cli.Flag{
+						commonOtlpFlags,
+						commonPartFlags,
+					}),
+					Action: createCommandWithT(traceImportRemoteAction),
+				},
+				{
+					Name:      "print-local",
+					Usage:     "Print traces in a local file to the console",
+					ArgsUsage: "<traces file>",
+					Action:    createCommandWithT(tracePrintLocalAction),
+				},
+				{
+					Name:  "print-remote",
+					Usage: "Print traces from a remote viam machine to the console",
+					Description: `
+In order to use the print-remote command, the machine must have a valid shell type service.
+Organization and location are required flags if using name (rather than ID) for the part.
```

Comment:
Would be nice to use `createUsageText` helper here (and in the other new methods) to get consistent usage text formatting. Also because currently, the usage text on `viam traces get-remote --help` includes `USAGE: viam traces get-remote [command options] [target]`, but it isn't clear from this text that `part` is in fact a required field.